apps/condo/domains/common/schema/fields.js

```diff
@@ -30,7 +31,31 @@ const SENDER_FIELD = {
     },
 }
 
+
+const ADDRESS_META_FIELD = {
```

Comment:
Maybe, we can add Ajv validation here?

Reply:
This field contains dump from third-party address suggestion service.
Since the format is from third-party, it cannot be validated.
And there is no use-case to validate it.